OpenOversight/tests/conftest.py

```diff
@@ -320,11 +320,11 @@ def session(db):
     session = scoped_session(session_factory=sessionmaker(bind=connection))
     db.session = session
 
-    yield session
-
-    transaction.rollback()
```

Comment:
The `transaction.rollback()` function call here was redundant. SQLAlchemy was already taking care of it for any of the tests that needed it (i.e., `test__uuid_uniqueness_constraint`, `test_redirect_add_salary`, `test_add_department__duplicate`).